pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1000,6 +1003,38 @@
           data=~
           complete=%.y
       ==
+    ::  +handle-logout: handles an http request for logging out
+    ::
+    ++  handle-logout
+      |=  [authenticated=? =request:http]
+      ^-  [(list move) server-state]
+      ::  whatever we end up doing, we always redirect to the login page
+      ::
+      =/  response=$>(%start http-event:http)
+        :*  %start
+            response-header=[307 ['location' '/~/login']~]
```

Comment:
There was a patch this morning about 303 vs 307. (I don't actually know which one is better.)

Reply:
Oops, good catch. 307 preserves the request type, whereas 303 becomes a GET request to the new location. Here, too, that's more appropriate.